docs/nconnect-build-from-source-code.md

```diff
@@ -0,0 +1,109 @@
+---
+id: nconnect-build-from-source
+title: Build nConnect from Source
+---
+
+## Preparation
+
+*If you already have git and golang compiler on your computer, skip to the next section.*
+
+To build nConnect from source, we need **git** tool and **go** compiler.
+If your PC or laptop doesn't have git and go installed, please use the following link to install them:
```

Comment:
No need to distinguish PC/laptop etc. Just use machine or computer is enough

Reply:
You should check the usage of laptop in docs and replace them too

Reply:
Yes, modified.